staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go

```diff
@@ -1279,8 +1278,7 @@ func TestListCompression(t *testing.T) {
 			t.Logf("%d: body: %s", i, string(body))
 			continue
 		}
-		// TODO: future, restore get links
-		if !selfLinker.called {
+		if utilfeature.DefaultFeatureGate.Enabled(features.RemoveSelfLink) == selfLinker.called {
 			t.Errorf("%d: never set self link", i)
```

Comment:
same comment (not that it matters that much)

Reply:
done